app/assets/stylesheets/govuk_publishing_components/components/_attachment.scss

```diff
@@ -96,8 +96,6 @@ $thumbnail-icon-border-colour: govuk-colour("mid-grey");
   }
 
   .gem-c-attachment__link.govuk-link {
-    word-break: break-word;
```

Comment:
I'm not sure how removing this line relates to this change, can you clarify?

Reply:
I'm not sure either, I think that was deleted by mistake (although I didn't notice any new problems). I'll revert it.